packages/nightly/src/lib/injected-nightly.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  SignedTransaction as NearSignedTransaction,
+  Transaction as NearTransaction,
+} from "near-api-js/lib/transaction";
+import { PublicKey as NearPublicKey } from "near-api-js/lib/utils";
+export interface NearAccount {
+  accountId: string;
+  publicKey: NearPublicKey;
+}
+export interface WalletAdapter {
+  account: NearAccount;
+  connected: boolean;
+  signTransaction: (
+    transaction: NearTransaction
+  ) => Promise<NearSignedTransaction>;
+  signAllTransactions: (
+    transaction: Array<NearTransaction>
+  ) => Promise<Array<NearSignedTransaction>>;
+  connect: (onDisconnect?: () => void) => Promise<NearAccount>;
+  disconnect: () => Promise<void>;
+}
+
+export declare class NightlyInjected {
+  near: NearNightly;
+  private readonly _nightlyEventsMap;
+  constructor();
+  invalidate(): void;
+}
+
+export declare class NearNightly {
+  account: NearAccount;
+  _onDisconnect: () => void;
+  private readonly _nightlyEventsMap;
+  constructor(eventMap: Map<string, (data: unknown) => unknown>);
+  connect(
+    onDisconnect?: () => void,
+    eagerConnect?: boolean
+  ): Promise<NearAccount>;
+  disconnect(): Promise<void>;
+  signTransaction: (
+    transaction: NearTransaction
+  ) => Promise<NearSignedTransaction>;
+  signAllTransactions: (
+    transaction: Array<NearTransaction>
+  ) => Promise<Array<NearSignedTransaction>>;
+}
```

Comment:
We should be able to simply define interfaces in this file instead of declaring classes:

```suggestion
export NightlyAccount {
  accountId: string;
  publicKey: NearPublicKey;
}

export interface NearNightly {
  account: NightlyAccount;
  connected: boolean;
  signTransaction: (transaction: NearTransaction) => Promise<NearSignedTransaction>;
  signAllTransactions: (transaction: Array<NearTransaction>) => Promise<Array<NearSignedTransaction>>;
  connect: (onDisconnect?: () => void, eagerConnect?: boolean) => Promise<NightlyAccount>;
  disconnect: () => Promise<void>;
}

export interface InjectedNightly {
  near: NearNightly;
  invalidate: () => void;
}
```

Reply:
Fixed https://github.com/near/wallet-selector/pull/314/commits/55b918ebf6991e08b32220b4197d9443c6ae760d